### ISSUE TITLE
How to disable generating test file automaitically?

### ISSUE THREAD

**Issue by livrth** [Jatana/FastOlympicCoding#53]
Sorry to open this stupid issue, but in most cases, I really don't need to do some test. In default config settings file,
I can't find any parameter to disable automatically generating test file after I compile and run.

My system is ubuntu20.04 LTS, and sublime version is Build 4113.

Thanks.

**Comment by Jatana** [comment]
Yeah, I think it is a good idea not to create a test file when there are no tests.

I'll fix that soon.
